bot/exts/fun/movie.py

```diff
@@ -12,17 +12,23 @@
 from bot.utils.extensions import invoke_help_command
 from bot.utils.pagination import ImagePaginator
 
+logger = logging.getLogger(__name__)
+
 # Define base URL of TMDB
 BASE_URL = "https://api.themoviedb.org/3/"
 
-logger = logging.getLogger(__name__)
+# Logo of TMDB
+THUMBNAIL_URL = "https://i.imgur.com/LtFtC8H.png"
 
 # Define movie params, that will be used for every movie request
 MOVIE_PARAMS = {
     "api_key": Tokens.tmdb,
     "language": "en-US"
 }
 
+# Define maximum value for `pages` param
+MAX_PAGES = 500
+
```

Comment:
I think it would be good to mention that this is different to what is documented
```suggestion
# Maximum value for `pages` API parameter. The maximum is documented as 1000 but
# anything over 500 returns an error.
MAX_PAGES = 500

```

Reply:
Yeah that makes sense.